helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/values.yaml

```diff
@@ -46,6 +46,8 @@ k8sMetadata:
     - k8s-app
     - release
 
+sourcetypePrefix: "kube:"
```

Comment:
Does it handle null values? (eg: "", ' ', <no value>, null)

Reply:
good catch. it changes them to default kube:. let me fix it up.

Reply:
Also, is ":" a standard separator? What if when prefix doesn't contain ":"?

Reply:
it doesn't have negative impact if prefix doesn't end with ":"

Reply:
Ok. May be change the default to  "kube" + ":" ?

Reply:
sounds good. updated.